tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckDates.java

```diff
@@ -780,22 +780,18 @@ public static String checkIso8601(String path, String value) {
         switch (key) {
             case "dateTimeFormatLength":
                 {
+                    // should be something like
+                    // ldml/dates/calendars/calendar[@type="gregorian"]/dateTimeFormats/dateTimeFormatLength[@type="full"]/dateTimeFormat[@type="standard"]/pattern[@type="standard"]
+                    // {1}, {0}
+
                     int index0 = value.indexOf("{0}");
                     int index1 = value.indexOf("{1}");
                     if (index0 < index1) {
-                        return "Field out of order: {0}…{1}";
+                        return "The {0} field (=time) cannot come before the {1} field (=date), in a YMD calendar.";
```

Comment:
Consider saying YMD (Year-first) or vice-versa

Reply:
```suggestion
                        return "In a YMD calendar, the {1} field must come before the {0} field.";
```

see comment on test below